extensions/twitch.js

```diff
@@ -0,0 +1,297 @@
+            },
+          },
+          {
+            opcode: "image",
+            text: "create image with image [image] with width [width] with height [height] with position [position] is bottom [bottom] is top [top]",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+            arguments: {
+              image: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "https://tekinical.github.io/twitchURI.png",
+              },
+              width: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "50px",
+              },
+              height: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "50px",
+              },
+              position: {
+                type: Scratch.ArgumentType.STRING,
+                defaultValue: "absolute",
+              },
+              bottom: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+              top: {
+                type: Scratch.ArgumentType.BOOLEAN,
+                defaultValue: false,
+              },
+            },
+          },
+          {
+            opcode: "closetext",
+            text: "close text",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            opcode: "closeimage",
+            text: "close image",
+            blockType: Scratch.BlockType.COMMAND,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            blockType: Scratch.BlockType.LABEL,
+            text: "Utilities",
+          },
+          {
+            opcode: "true",
+            text: "true",
+            blockType: Scratch.BlockType.BOOLEAN,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+          {
+            opcode: "false",
+            text: "false",
+            blockType: Scratch.BlockType.BOOLEAN,
+            blockIconURI: "https://tekinical.github.io/twitchURI.png",
+          },
+        ],
+      };
+    }
+
+    setupchat(args) {
+      var Link =
+        "https://nightdev.com/hosted/obschat/?theme=bttv_dark&channel=" +
+        args.username +
+        "&fade=" +
+        args.fade +
+        "&bot_activity=" +
+        args.bots +
+        "&prevent_clipping=" +
+        args.prevent;
+      chat = document.createElement("iframe");
+      chat.style.width = args.width;
+      chat.style.height = args.height;
+      chat.style.border = "none";
+      chat.style.position = args.position;
+      chat.setAttribute("allowtransparency", "true");
+      chat.setAttribute("src", Link);
+      Scratch.renderer.addOverlay(chat, "manual");
+    }
+
+    closechat() {
+      if (chat == null) {
+        console.log("Can't close chat if chat wasn't opened.");
```

Comment:
I don't think this console log is necessary.

Reply:
Changed to alert.

Reply:
Please remove console logs/alerts, they're unnecessary in extensions